src/scss/components/_dynamic_list.scss

```diff
@@ -0,0 +1,56 @@
+.dynamic-list {
+  &__title {
+    font-size: 21px;
+    line-height: 28px;
+    font-weight: 400;
+  }
+
+  &__container {
+    overflow: auto;
+  }
+}
+
+.dynamic-list-item {
+  display: flex;
+  list-style: none;
+  padding: $magicnumber1;
```

Comment:
What does `magicnumber1` refer to? Could it be renamed to something more meaningful? 😄

Reply:
I have no idea. 
This is an awful variable name but seems to be used throughout Florence not something I have added. but a lot of padding margins and even text heights have been set with it.

Reply:
Fair, may be worth changing the name to base or something, but I won't hold you to it